plugins/themes/default/styles/objects/article_details.less

```diff
@@ -10,6 +10,10 @@
  */
 .obj_article_details {
 
+	h2 {
+		margin: 0;
+	}
+
```

Comment:
I think you can remove this because every `<h2>` should have a class of `label` or `subtitle` with its margins set there.